src/Db/Entity/Toggle.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace Clearbooks\Labs\Db\Entity;
+
+class Toggle extends CamelCaseMapperEntity
```

Comment:
OK leave it then.
The only reason I'm saying this is because toggle class ties together a lot of tests, meaning a lot of responsibilities.
Like ToggleTest kind of testing CamelCaseMapperEntity constructor.